content/docs/accessibility.md

```diff
@@ -217,6 +217,120 @@ initially triggered the modal.
 >While this is a very important accessibility feature, it is also a technique that should be used judiciously. Use it to repair the keyboard focus flow when it is disturbed, not to try and anticipate how
 >users want to use applications.
 
+## Mouse and pointer events
+
+Ensure that all functionality exposed through a mouse or pointer event can also be accessed using the keyboard alone. Depending only on the pointer device will lead to many cases where
+keyboard users cannot use your application.
+
+To illustrate this, let's look at a prolific example of broken accessibility caused by click events. This is the outside click pattern, where a user can disable an opened popover by clicking outside the element. 
+
+<img src="../images/docs/outerclick-with-mouse.gif" alt="A toggle button opening a popover list implemented with the click outside pattern and operated with a mouse showing that the close action works." />
+
+This is typically implemented by attaching a `click` event to the `window` object that closes the popover:
+
+```javascript{12-16,27}
+class OuterClickExample extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { isOpen: false };
+    this.toggleContainer = createRef();
+
+    this.onClickHandler = this.onClickHandler.bind(this);
+  }
+
+  componentDidMount() {
+    window.addEventListener('click', event => {
+      if (this.state.isOpen && !this.toggleContainer.current.contains(event.target)) {
+        this.setState({ isOpen: false });
+      }
+    });
+  }
+
+  onClickHandler() {
+    this.setState({
```

Comment:
Since it depends on the current state, a better idea is to use the:
```js
this.setState(current_state => {
  isOpen: !current_state.isOpen
})
```
style of `setState` invocation.

Reply:
Changed.